src/org/apache/cocoon/transformation/SparqlTransformer.java

```diff
@@ -317,12 +319,12 @@ private void executeRequest(String url, String method, Map<String, String> httpH
           AttributesImpl attrs = new AttributesImpl();
           attrs.addCDATAAttribute("status", ""+responseCode);
           xmlConsumer.startElement(SPARQL_NAMESPACE_URI, "error", "sparql:error", attrs);
-          String responseBody = httpMethod.getStatusText(); //httpMethod.getResponseBodyAsString();
-          xmlConsumer.characters(responseBody.toCharArray(), 0, responseBody.length());
+          responseBody = httpMethod.getResponseBodyAsString();
+          statusText = httpMethod.getStatusText();
+          xmlConsumer.characters(responseBody.toCharArray(), 0, statusText.length());

```

Comment:
DIT KAN NIET!
Je neemt de lengte van `statusText` en je haalt de tekst uit `responseBody`? AYKM?
Dit is een gegarandeerde `ArryIndexOutOfBoundsException`.